apps/src/templates/projects/ProjectCardGrid.jsx

```diff
@@ -86,14 +86,13 @@ class ProjectCardGrid extends Component {
             <ProjectAppTypeArea
               labKey="featured"
               labName={i18n.featuredProjects()}
-              labViewMoreString={i18n.projectTypeDanceViewMore()}
               projectList={projectLists.featured}
               numProjectsToShow={numProjects}
               galleryType={this.props.galleryType}
               navigateFunction={this.onSelectApp}
               isDetailView={false}
               hideViewMoreLink={true}
-              hideWithoutThumbnails={true}
```

Comment:
Does this change have any potential impact on other lab types?

Reply:
Since the public gallery only displays curated featured projects, admin will select projects for which thumbnail images are available. At '/projects/featured' (the admin featured projects page), the thumbnail images are displayed alongside a link to the project.

I believe in the past, since any project could be published, this was a way to filter projects without thumbnail images from being included in the public gallery (projects without any content).
As new labs are added, either they'll have a way to generate a corresponding thumbnail OR we we'll need to provide a thumbnail image as we are doing for music lab. To my knowledge, legacy labs that are publishable generate thumbnail images.

Reply:
Makes sense, thanks!